Mathlib/NumberTheory/ModularForms/CongruenceSubgroups.lean

```diff
@@ -18,33 +18,33 @@ It also contains basic results about congruence subgroups.
 
 -/
 
-
-local notation "SL(" n ", " R ")" => Matrix.SpecialLinearGroup (Fin n) R
-
 attribute [-instance] Matrix.SpecialLinearGroup.instCoeFun
 
-local notation:1024 "↑ₘ" A:1024 => ((A : SL(2, ℤ)) : Matrix (Fin 2) (Fin 2) ℤ)
-
 open Matrix.SpecialLinearGroup Matrix
 
 variable (N : ℕ)
 
 local notation "SLMOD(" N ")" =>
   @Matrix.SpecialLinearGroup.map (Fin 2) _ _ _ _ _ _ (Int.castRingHom (ZMod N))
+namespace CongruenceSubgroup
 
+scoped notation "SL(" n ", " R ")" => Matrix.SpecialLinearGroup (Fin n) R
```

Comment:
It turns out that this notation already exists, scoped in the `MatrixGroups` namespace.